BUILDING.md

```diff
@@ -253,14 +292,21 @@ can be made about any sort of ABI stability across the same SONAME version.
 
 ## Building with make on UNIX systems (deprecated)
 
-NOTE: `make` used to be gRPC's default build system, but we're no longer recommending it. You should use `bazel` or `cmake` instead. The `Makefile` is only intended for internal usage and is not meant for public consumption.
+> **Warning**
+> Building gRPC using `make` is no longer recommended. 
```

Comment:
Again, this is not really a warning, it's basic usage and support. Calling out a separate Note about the makefile is also rather unnecessary. I'd recommend leaving this paragraph mostly as is.

Reply:
@veblush @jtattermusch I vote to just delete this section. Objections?

Reply:
I've reverted to the original text.

Reply:
I'm happy to delete this section in the PR. Please confirm if the contained section (i.e. `A note on protoc`) can also be removed?